src/app/services/class.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
```

Comment:
This service is very redundant. You can have a subscription service that fetches your subscriptions along with adding subscriptions.

Reply:
Good point, service has been removed and its functionality was merged to the subscription service.